### PR TITLE
Feature/expose default traversability param

### PR DESCRIPTION
**Pull request by marco-tranzatto**
This PR allows to set/get/restore the param `traversability_default`.

By changing this parameter, it is possible to change the behavior of a higher level planner about unknown regions in a traversability map. The more `traversability_default` is closer to 0, the less unknown regions are considered when planning. Vice-versa for `traversability_default` closer to 1.

Given parameter for `traversability_default` is checked against constant bounds to make sure it's between 0 and 1.

### CODE REVIEW

File: traversability_estimation/include/traversability_estimation/TraversabilityMap.hpp [hunk 1]
```diff
@@ -129,6 +133,23 @@ class TraversabilityMap
...
+   * Gets the default traversability value of unknown regions in the map.
+   * @return default traversability value of unknown regions in the map
+   */
+  double getDefaultTraversabilityUnknownRegions() const;
+
+  /*!
+   * Sets the default traversability value of unknown regions in the map.
+   * @param[in] defaultTraversability new default traversability value of unknown regions in the map
+    */
+  void setDefaultTraversabilityUknownRegions(const double& defaultTraversability);
```
Comment (remod) [inline traversability_estimation/include/traversability_estimation/TraversabilityMap.hpp]: Typo (sorry, I get notified by all PRs where I am maintainer from the ANYbotics side ;) )

Comment (marco-tranzatto) [reply traversability_estimation/include/traversability_estimation/TraversabilityMap.hpp]: I feel stupid, but I cannot spot the typo

Comment (lorenwel) [reply traversability_estimation/include/traversability_estimation/TraversabilityMap.hpp]: Missing "n" in "Unknown".

**Comment by marco-tranzatto** [comment]
Auto approving my last commit to merge the PR feels like this:

approve_your_own_prs

**Pull request merged**